src/lib/rate-limiting.ts

```diff
@@ -54,29 +67,20 @@ const getStore = _.once(() => {
 	// the whole process
 	client.on('error', _.throttle(logRedisError, 300000));
 
-	const opts: ExpressBruteRedisOpts = {
-		client,
-		prefix: 'api:ratelimiting:',
-	};
-
-	const redisStore = new ExpressBruteRedis(opts);
-	return new FacadeStore(redisStore);
-});
-
-// If redis system is not working, we bypass the rate limiting, to not
-// block the functionality of the API.
-const redisErrorHandler = (options: StoreErrorOptions) => {
-	options.next();
-};
-
-const failDebug: ExpressBrute.FailTooManyRequests = (
-	req,
-	res,
-	next,
-	nextValidRequestDate,
-) => {
-	console.error('Blocked by rate limiting: ' + req.originalUrl);
-	ExpressBrute.FailTooManyRequests(req, res, next, nextValidRequestDate);
+	// const opts: ExpressBruteRedisOpts = {
+	// 	client,
+	// 	prefix: 'api:ratelimiting:',
+	// };
+
+	// const redisStore = new ExpressBruteRedis(opts);
+	return new RateLimiterRedis({
```

Comment:
Is there a way to have a key prefix for the redis backend as well? We should try to namespace redis.